x/evm/keeper/msg_server.go

```diff
@@ -280,7 +280,9 @@ func (s msgServer) Link(c context.Context, req *types.LinkRequest) (*types.LinkR
 			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
 			sdk.NewAttribute(types.AttributeKeyChain, senderChain.Name),
 			sdk.NewAttribute(types.AttributeKeyBurnAddress, burnerAddr.Hex()),
+			sdk.NewAttribute(types.AttributeKeyDepositAddress, burnerAddr.Hex()),
```

Comment:
`burnerAddr` is already there in the line above. Why do you need another key for the same value?

Reply:
definitely

Reply:
ok done, just changing to depositAddress